main.go

```diff
@@ -102,6 +102,13 @@ func main() {
 	hc := NewHealthChecker(ds)
 	hcs := &HealthCheckWebService{healthChecker: hc}
 
+	file, err := os.OpenFile("/var/log/pgreba/pgreba.log", os.O_CREATE|os.O_APPEND|os.O_WRONLY, 0644)
```

Comment:
This will still cause the disk to fill up because the app with continue to write to this fd unless truncate is used. What are your thoughts around using gotee to point stdout to a specific file where SIGHUP signals a fd change?

https://github.com/quixoten/gotee

If we want to make logging to a file a first-class option, that's totally fine too. Just wanted to look at systemd service options or gotee before adding since there are lots of ways to shave this yak.